sunxspex/tests/test_brem.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+
+
+from sunxspex import emission
+
+
+def test_brem_collisional_loss():
+    photon_energies = np.array([1.0, 10.0, 100.0, 1000.0])
+    res = emission.collisional_loss(photon_energies)
+    # IDL code to generate values taken from dedt variable
+    # Brm_ELoss, [1.0, 10.0, 100.0, 1000.0], dedt
+    # print, dedt, format='(e0.116)'
+    assert np.array_equal(res, [3.6275000635609979e+02, 1.2802678240553834e+02,
```

Comment:
Do these constants force double precision arithmetic when our knowledge doesn't begin to punch below a percent?  I'm not a fan of unnecessary DP arithmetic for computations that don't need it and which would slow down repeated computations.